src/__tests__/regression/read-state.test.ts

```diff
@@ -49,13 +51,13 @@ describe('readState', () => {
     const contractTxId = 'CbGCxBJn6jLeezqDl1w3o8oCSeRCb-MmtZNKPodla-0';
     const blockHeight = 707892;
     const result = await readContract(arweave, contractTxId, blockHeight);
-    const resultString = JSON.stringify(result).trim();
+    const resultString = stringify(result).trim();
```

Comment:
for the `t9T7DIOGxx4VWXoCEeYYarFYeERTpWIC1V3y-BPZgKE` contract the JSON.stringify turned to be non-deterministic.